src/style/aframe.css

```diff
@@ -28,10 +28,37 @@ body {
 
 a-assets,
 a-scene video,
-a-scene img {
+a-scene img,
+a-scene audio {
   display: none;
 }
 
+.a-enter-vr-modal,
+.a-orientation-modal {
+  font-family: Andale Mono, Courier New, monospace;
+}
+
+.a-enter-vr-modal {
+  font-size: 11px;
+  line-height: 17px;
```

Comment:
might want to reduce `line-height` so the shortest height doesn't pretrude by a few pixels